checkov/terraform/checks/resource/ncp/NKSPublicAccess.py

```diff
@@ -0,0 +1,23 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class NKSPublicAccess(BaseResourceCheck):
```

Comment:
you actually want to use negative resource value check here. Here is an example of how it looks like https://github.com/bridgecrewio/checkov/blob/master/checkov/terraform/checks/resource/aws/DMSReplicationInstancePubliclyAccessible.py